### ISSUE TITLE
LiveTL interface needs a way to reload chat

### ISSUE THREAD

**Issue by kmarple1** [LiveTL/LiveTL#164]
I'm not sure why, but YouTube chat frequently freezes for me. On YouTube, hiding and showing the chat fixes it. However, in the LiveTL interface, there doesn't seem to be a way to do this other than refreshing the page. I don't know how common this problem is, but it would be nice to have a button to refresh just the chat.

**Comment by KentoNishi** [comment]
In the meantime you can right click the chat and click "Reload Frame"

**Comment by BagdanGilevich** [comment]
Yes I also experience this bug in YouTube (am using latest Firefox) when watching stream archives

**Comment by Hughp135** [comment]
Is this related to having hyperchat enabled? Created issue at https://github.com/LiveTL/chat/issues/8
